packages/expo-file-system/src/next/FileSystem.types.ts

```diff
@@ -0,0 +1,98 @@
+/**
+ * A string representing a file or directory path.
+ */
+export type Path = string;
+
+/**
+ * Represents a directory on the file system.
+ */
+export declare class Directory {
+  /**
+   * Creates an instance of a directory.
```

Comment:
This may sound like it creates the actual directory. Probably worth emphasizing it's not the case. It creates a representation of a directory with the given path.